src/main/java/io/fusionauth/client/FusionAuthClient.java

```diff
@@ -311,6 +311,22 @@ public FusionAuthClient setTenantId(UUID tenantId) {
         .go();
   }
 
+  /**
+   * Creates an authenticatio API key. You can optionally specify an Id for the application, if not provided one will be generated.
+   *
+   * @param authenticationKeyId (Optional) The Id of the authentication key. If not provided a secure random UUID will be generated.
+   * @param request The request object that contains all of the information used to create the APIKey.
+   * @return The ClientResponse object.
+   */
+  public ClientResponse<APIKeyResponse, Errors> createAPIKey(String authenticationKeyId, APIKeyRequest request) {
```

Comment:
Should there be a bunch more APIs here? 

- Retrieve API key by Id
- Retrieve API keys
- Delete API key
- Update API key
- PATCH API key

This will be in the client builder.